opentelemetry-sdk/src/opentelemetry/sdk/resources/__init__.py

```diff
@@ -141,25 +141,32 @@
 class Resource:
     """A Resource is an immutable representation of the entity producing telemetry as Attributes."""
 
-    def __init__(self, attributes: Attributes):
+    def __init__(self, attributes: Attributes, schema_url: str):
```

Comment:
This should be an optional parameter as to not change the signature of the constructor, otherwise this would be a breaking change.

Reply:
Done! I'm wondering whether I should change the constructor to make its behavior more consistent with the factory method.